src/components/form-elements/radio/_radio.scss

```diff
@@ -142,6 +142,7 @@ $radioCheckedBorderWidth: 5px;
     height: $radioSize;
     border-radius: 50%;
     box-shadow: 0px 0px 0px var(--radioBorderWidth) var(--radioRingColor) inset;
+    transition: box-shadow 0s;
```

Comment:
Why do we need this?

Reply:
We don't have exit animation on the radio (when radio becomes unchecked), so I had to disable transition on the ring thickness change.